Cargo.toml

```diff
@@ -28,6 +28,9 @@ vmm-sys-util = "0.6.0"
 vm-virtio = { path = "vm-virtio" }
 vhost_rs = { git = "https://github.com/cloud-hypervisor/vhost", branch = "dragonball", package = "vhost", features = ["vhost-user-slave"] }
 
+[patch.crates-io]
+vm-memory = { git = "https://github.com/cloud-hypervisor/vm-memory", branch = "ch" }
```

Comment:
Oh so we use the `patch` property of the Cargo.toml because we expect this to be quickly fixed in rust-vmm/vm-memory?

Reply:
So we don't have to fork all the repos that use vm-memory.